app-misc/qlcplus/qlcplus-4.11.1.ebuild

```diff
@@ -6,7 +6,7 @@ EAPI=6
 inherit qmake-utils udev vcs-snapshot virtualx xdg
 
 DESCRIPTION="QLC+ - Q Light Controller Plus to control DMX interfaces"
```

Comment:
description is also wrong - it containts a package name

Reply:
thanks for the hint, fixed.